account_due_list/README.rst

```diff
@@ -0,0 +1,68 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
change to .png

Reply:
I have changed it... but in already migrated modules in this repo is .svg and I can't find any related annotation in *Migration-to-version-11.0*